index.html

```diff
@@ -183,7 +191,6 @@ <h2 class="title-post1">Profesional Art Printing Data</h2>
                             </ul>
                         </nav>
                     </div>
-                    <button class="see-project-btn1">See Project</button>
                 </div>
             </div>
```

Comment:
- [ ] Kindly check your `about me` section, at the moment, there's a `padding` or `margin` to both sides of the section in both desktop and mobile views. I have attached some images to make you see this better, fixing this will match the design and make your work look nicer :point_down: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/214163339-bc464290-5831-4750-ab92-774c02fff9c2.png)|![image](https://user-images.githubusercontent.com/99715640/214163447-64e8363e-c538-4ff0-9989-76e61d10d9c0.png)|